build/scripts/DownloadDotNetRuntimeInstaller.ps1

```diff
@@ -0,0 +1,43 @@
+[CmdLetBinding()]
+Param(
+    [string]$Platform = "x64",
+    [string]$OutputDirectory = ""
```

Comment:
Why default it? Why not

```
[string]$OutputDirectory
```

Won't `Join-Path $null "foo"` work equally well as `Join-Path "" "foo"` ?

Reply:
Just found out that Join-Path doesn't work with null OR empty string. Making these params required.

Reply:
> Just found out that Join-Path doesn't work with null OR empty string

Sure does

`$meh = Join-Path "$foo" "$bar"`

Then it doesn't matter if $foo or $bar are $null or "" :-)